docs/src/main/asciidoc/all-builditems.adoc

```diff
@@ -7,7 +7,8 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 include::./attributes.adoc[]
-
+:categories: writing-extensions
+:summary: Explore all the BuildItems you can consume/produce in your extensions.
 = Build Items
```

Comment:
The title here needs to move up to line 9, doesn't it?

Reply:
It doesn't help with the index file generation, but sure, for consistency with the rest of the files.